WcaOnRails/config/locales/es.yml

```diff
@@ -1401,6 +1401,7 @@ es:
       delegates: Delegados de la WCA
       #original_hash: 37b8fdc
       organizations: Organizaciones Nacionales
+      forum: Foro
```

Comment:
cc @Ivan-Brigidano: same "issue", due to how our tests are written, when adding an item to the navigation we need the French and Spanish translation for it :s
Would you mind confirming or providing a suitable translation please?

Reply:
"Foro" sounds good!

Reply:
Thanks!